Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         }
         
         eventDetailViewController.theEvent = event;
-        //eventDetailViewController.title = event.title;
```

Comment:
There's a bunch of commented code in the CalendarsViewController.m and a bit in the EventDetailsViewController.m that I haven't touched and wasn't sure if it needed to stay or if I should delete it all.